explainerdashboard/dashboards.py

```diff
@@ -1030,11 +1036,8 @@ def __init__(self, dashboards:List[ExplainerDashboard], title:str="ExplainerHub"
             self.app.config['SECRET_KEY'] = secret_key
         SimpleLogin(self.app, login_checker=self._validate_user)
         
-        self.base_route = base_route

```

Comment:
Should we keep this line ?

Reply:
No need. `self._store_params(no_store=['dashboards', 'logins', 'secret_key'])` automatically stores all parameters to attributes. (saves you a bunch of boilerplate)

Reply:
Nice !